src/BARON.jl

```diff
@@ -185,13 +196,15 @@ function to_str(c::Expr)
             else
                 return wrap_with_parens(string(c.args[1], wrap_with_parens(to_str(c.args[2]))))
             end
+        else
+            throw(UnrecognizedExpressionException(c))
```

Comment:
Could you change the message to something like ``"unrecognized comparison expression: $c``?